R/estimate_vaccineff.R

```diff
@@ -91,38 +98,32 @@ effectiveness <- function(vaccineff_data,
     end_cohort = vaccineff_data$end_cohort
   )
   # Estimation at
-  eff_obj$at <- at
-  # effectiveness object
-  class(eff_obj) <- "effectiveness"
+  vaccineff$at <- at
+  # vaccineff object
+  class(vaccineff) <- "vaccineff"
 
   ## return
-  return(eff_obj)
+  return(vaccineff)
 }
 
 #' @title Summarize VE Results
 #'
-#' @description Summarizes the results of `effectiveness`.
+#' @description Summarizes the results of `vaccineff`.
 #'
-#' @param object Object of the class `effectiveness`.
+#' @param object Object of the class `vaccineff`.
 #' @param ... Additional arguments passed to other functions.
-#' @return Summary of the results from effectiveness.
+#' @return Summary of the results from `estimate_vaccineff`.
 #' @export
 
-summary.effectiveness <- function(object, ...) {
-  # Check if the input object is of class "effectiveness"
-  stopifnot("Input must be an object of class 'effectiveness'" =
-      checkmate::test_class(object, "effectiveness")
+summary.vaccineff <- function(object, ...) {
+  # Check if the input object is of class "vaccineff"
+  stopifnot("Input must be an object of class 'vaccineff'" =
+      checkmate::test_class(object, "vaccineff")
+  )
+  cat(
+    sprintf("Vaccine Effectiveness at %i days computed as VE = 1 - HR:\n",
+            object$at)
   )
-  if (!is.null(object$at)) {
-    cat(
-      sprintf("Vaccine Effectiveness at %i days computed as VE = 1 - HR:\n",
-              object$at)
-    )
-  } else {
-    cat(
-      "Vaccine Effectiveness at the end of study computed as VE = 1 - HR:\n"
-    )
-  }
```

Comment:
Could you confirm this is no longer relevant? Goes beyond renaming so double-check 😊

Reply:
Yes, It's not necessary anymore, since `at` must be provided to make the estimation fully meaningful. But it's a good sanity check, we also hesitated a bit about this within the team :)